src/cmd-virt-install

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+
+# "install" a coreos system with libvirt.  This code in the future
+# ideally gets added to `virt-install`.  See also https://github.com/coreos/fedora-coreos-tracker/issues/235
+# It is inspired by https://coreos.com/os/docs/latest/booting-with-libvirt.html
+
+import argparse
+import subprocess
+import json
+import os
+import re
+import sys
+import random
+import string
+import tempfile
+
+import libvirt
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, cosa_dir)
+
+from cosalib import cmdlib
+from cosalib.builds import Builds
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--connect", "-c", help="libvirt URI", default="qemu:///system")
+parser.add_argument("--ignition", "-i", help="Ignition config", required=True)
+parser.add_argument("--build", help="Build ID")
+parser.add_argument("--pool", help="storage pool", default="default")
+parser.add_argument("--name", help="Prefix for libvirt resources")
+parser.add_argument("--instid", help="Suffix appended to libvirt resources")
+parser.add_argument("vinstall_args", nargs='*', help="Additional arguments for virt-install")
```

Comment:
Might want to use the `parse_known_args` approach here too so we don't hit the same issue that prompted https://github.com/coreos/coreos-assembler/pull/1058.

Reply:
Done!